mlir/lib/Dialect/Tensor/IR/TensorOps.cpp

```diff
@@ -10,7 +10,7 @@
 #include "mlir/Dialect/Arith/IR/Arith.h"
 #include "mlir/Dialect/Arith/Utils/Utils.h"
 #include "mlir/Dialect/Complex/IR/Complex.h"
-#include "mlir/Dialect/Linalg/IR/Linalg.h"
+#include "mlir/Dialect/Linalg/IR/LinalgRelayoutInterface.h.inc"
```

Comment:
I would prefer not to include `.inc` files from other dialects.

Let me try to create a separate bazel target for LinalgInterfaces.h, then we don't need any cmake changes and can include LinalgInterfaces.h instead.

Reply:
Agree, sorry I missed that.

Reply:
You shouldn't include a .h.inc, but you can create a `include "mlir/Dialect/Linalg/IR/LinalgRelayoutInterface.h"` that itself encapsulates and include the .h.inc.
See for example `include/mlir/Dialect/Bufferization/IR/AllocationOpInterface.h`, `include/mlir/Dialect/GPU/IR/CompilationInterfaces.h`, or `include/mlir/Conversion/ConvertToLLVM/ToLLVMInterface.h`  which implement this pattern.